strongbox-data-service/src/test/resources/etc/conf/janusgraph-inmemory-single-parition.properties

```diff
@@ -0,0 +1,10 @@
+storage.backend = inmemory
```

Comment:
The name of this file is incorrect. It should be called `janusgraph-inmemory-single-partition.properties `.

Reply:
fixed